conf.json

```diff
@@ -0,0 +1,11 @@
+{
+  "data_dir": ".",
+  "sim_file": "sim.json",
+  "total_time": null,
+  "print_batch_size": 500,
+  "log_level": "info",
+  "expected_pmt_amt": 3800000,
+  "capacity_multiplier": 2,
+  "no_results": false,
+  "log_interval": 60
+}
```

Comment:
Slight personal preference to use a INI file over json, _but_ we already have `json` in the project so not a strongly held opinion. If it's easy to switch over to serde (suggested in comments that follow), would like to consider it if other folks feel the same.

Reply:
Thought to use json because we already use it but happy to switch to INI.